auto_scheduler/pom.xml

```diff
@@ -20,8 +20,8 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
[java stream ](https://docs.oracle.com/javase/8/docs/api/java/util/stream/package-summary.html) are available in java 8 and later. so need to upgrade the version of java being used.